modules/apps/27-interchain-accounts/types/packet.go

```diff
@@ -47,6 +51,83 @@ func (iapd InterchainAccountPacketData) GetBytes() []byte {
 	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&iapd))
 }
 
+/**
+
+ADR-8 CallbackPacketData implementation
```

Comment:
Is this the right place for this documentation? Should I instead have it in a README in ICS27 directory?

Reply:
it might make sense to have it in the ICS27 directory _as well_, but I think it would be nice to have here as well so there is an example of how to make use of this ADR-8

Reply:
I believe we don't really make use of README files in our different modules, so maybe it's better to add this to our regular docs site?

Reply:
I think in-code and external facing documentation (information under `docs/`) are great. The in-code documentation should help explain the hard-coded values while the external facing documentation should explain how to interact with it